src/org/labkey/targetedms/TargetedMSSchema.java

```diff
@@ -1582,6 +1595,30 @@ public DisplayColumn createRenderer(ColumnInfo colInfo)
         {
             return new QCEmailNotificationsTable(this, cf);
         }
+        if (TABLE_MS_PROJECT.equalsIgnoreCase(name) ||
+                TABLE_MS_INSTRUMENT.equalsIgnoreCase(name) ||
+                TABLE_PAYMENT_METHOD.equalsIgnoreCase(name) ||
+                TABLE_PROJECT_PAYMENT_METHOD.equalsIgnoreCase(name) ||
+                TABLE_INSTRUMENT_SCHEDULE.equalsIgnoreCase(name))
+        {
+            var result = new FilteredTable<TargetedMSSchema>(getSchema().getTable(name), this, cf)
+            {
+                @Override
+                public boolean hasPermission(@NotNull UserPrincipal user, @NotNull Class<? extends Permission> perm)
+                {
+                    return (getContainer().hasPermission(user, AdminPermission.class));
```

Comment:
Why is this override needed at all? This will prevent non-admins from accessing the table.